### PR TITLE
remove the about dialog

### PR DESCRIPTION
**Pull request by danrabbit**
- [x] After #124 all information in the About dialog will be present on the AppInfo page in AppCenter

### CODE REVIEW

**Comment by donadigo** [comment]
Hm, can't we e.g: make this about menuitem open the AppCenter with the app info rather than removing it completely? Just suggesting.

**Comment by danrabbit** [comment]
I would specifically like to remove the menu item because it's cluttering up slingshot search

File: data/org.pantheon.scratch.desktop.in [hunk 1]
```diff
@@ -10,7 +10,6 @@ Categories=GNOME;GTK;IDE;Utility;TextEditor;WebDevelopment;
 _Keywords=IDE;
 MimeType=text/plain;
 StartupNotify=true
-Actions=NewWindow;NewDocument;AboutDialog;
```
Comment (cassidyjames) [inline data/org.pantheon.scratch.desktop.in]: Did you mean to completely remove this key? I would think you would just want to remove the `AboutDialog;` bit for the smallest change.

Comment (danrabbit) [inline data/org.pantheon.scratch.desktop.in]: Oops you're right. My bad

**Pull request merged**